hikari/files.py

```diff
@@ -1072,6 +1142,22 @@ def stream(
         """
         return _NoOpAsyncReaderContextManagerImpl(IteratorReader(self.filename, self.mimetype, self.data))
 
+    async def save(
+        self,
+        path: Pathish,
+        *,
+        executor: typing.Optional[concurrent.futures.Executor] = None,
+        force: bool = False,
+    ) -> None:
+        if not isinstance(self.data, bytes):
```

Comment:
Could extend this to special case for bytearray and memoryview

Reply:
Expanded :)